tensorflow_hub/tools/make_image_classifier/make_image_classifier_lib.py

```diff
@@ -224,15 +225,19 @@ def train_model(model, hparams, train_data_and_size, valid_data_and_size):
       metrics=["accuracy"])
   steps_per_epoch = train_size // hparams.batch_size
   validation_steps = valid_size // hparams.batch_size
+  callbacks = []
+  if log_dir != None:
+    callbacks.append(tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1))
   return model.fit(
       train_data,
       epochs=hparams.train_epochs,
       steps_per_epoch=steps_per_epoch,
       validation_data=valid_data,
-      validation_steps=validation_steps)
+      validation_steps=validation_steps,
+      callbacks=callbacks)
 
 
-def make_image_classifier(tfhub_module, image_dir, hparams,
+def make_image_classifier(tfhub_module, image_dir, hparams, log_dir=None,
```

Comment:
Please add the new argument to the Args: section of the docstring.

Reply:
Should be okay. I used docstring I wrote for train_mode().